Remote Storage for Commerce Cloud additions

rvr31 opened the pull request:
## Purpose of this pull request
This pull request is adding missing documentation of enabling remote storage for Adobe Commerce Cloud.

## Affected pages
https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/storage/remote-storage/cloud-support.html

rvr31 committed:
Updated remote storage configuration for Cloud

Added information for setting the environment variables to enable remote storage using AWS S3 buckets.

rvr31 committed:
Added syncing logic to cloud remote storage.

Key information to sync the current files to the new remote storage location.

rvr31 committed:
Cloud Remote storage log information

Added information on the log line that is in the deployment logs

shrielenee commented on help/configuration/remote-storage/cloud-support.md:
````diff
@@ -5,6 +5,50 @@ description: See guidance on how to set up remote storage for Adobe Commerce on
+magento-cloud variable:create --level environment --name REMOTE_STORAGE --json true --inheritable false --value '{"driver":"aws-s3","prefix":"uat","config":{"bucket":"aws-bucket-id","region":"eu-west-1","key":"optional-key","secret":"optional-secret"}}'
+```
+
+Running this command will create a new environment variable called REMOTE_STORAGE with the specified JSON configuration. Alternatively, you could use the web interface to add the variable to the appropriate environment.
````
```suggestion
Running this command will create a new environment variable called `REMOTE_STORAGE` with the specified JSON configuration. Alternatively, you could use the web interface to add the variable to the appropriate environment.
```

shrielenee commented on help/configuration/remote-storage/cloud-support.md:
```diff
@@ -5,6 +5,50 @@ description: See guidance on how to set up remote storage for Adobe Commerce on
+
+The `key` and `secret` are optional. During the creation of the variable, it is possible to set it to `sensitive` so that the value is not visible in the web interface hiding the `key` and `secret`. If you choose not to use the `key` and `secret` authentication method, you need to ensure the server is authorized to the S3 bucket by other means. In that case, the `key` and `secret` should be omitt ...
+
+## Syncing the remote-storage
```
```suggestion
## Syncing the remote-storage

```

meker12 commented on help/configuration/remote-storage/cloud-support.md:
```diff
@@ -5,6 +5,50 @@ description: See guidance on how to set up remote storage for Adobe Commerce on
+The `key` and `secret` are optional. During the creation of the variable, it is possible to set it to `sensitive` so that the value is not visible in the web interface hiding the `key` and `secret`. If you choose not to use the `key` and `secret` authentication method, you need to ensure the server is authorized to the S3 bucket by other means. In that case, the `key` and `secret` should be omitt ...
+
+## Syncing the remote-storage
+After enabling the remote store module make sure to syncronize the current media files to the remote store location. 
```
```suggestion
After enabling the remote store module make sure to synchronize the current media files to the remote store location. 
```

meker12 commented on help/configuration/remote-storage/cloud-support.md:
```diff
@@ -5,6 +5,50 @@ description: See guidance on how to set up remote storage for Adobe Commerce on
+
+## Syncing the remote-storage
+After enabling the remote store module make sure to syncronize the current media files to the remote store location. 
+To start the syncronisation run the command below on the enviroment where you enabled the remote storage.
```
```suggestion
To start the synchronization run the command below on the environment where you enabled the remote storage.
```

rvr31 committed:
Apply suggestions from code review

Co-authored-by: Shrie Spangler <sspangler@magento.com>
Co-authored-by: Margaret Eker <meker@adobe.com>

rvr31 committed:
Update help/configuration/remote-storage/cloud-support.md

Co-authored-by: Margaret Eker <meker@adobe.com>

meker12 reviewed (commented):
Maybe need to double-check this info.

In the [Cloud release notes](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/release-notes/ece-tools-package.html), it specifically says that the REMOTE_STORAGE variable is not supported on Cloud infrastructure.

And the remote storage config instructions point to another [topic](https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/storage/remote-storage/cloud-support.html?lang=en) that explains how to configure remote storage using Fastly.

hguthrie reviewed (changes requested):
This requires a technical review, so blocking for now.

rvr31 commented:
> Maybe need to double-check this info.
> 
> In the [Cloud release notes](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/release-notes/ece-tools-package.html), it specifically says that the REMOTE_STORAGE variable is not supported on Cloud infrastructure.
> 
> And the remote storage config instructions point to another [topic](https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/storage/remote-storage/cloud-support.html?lang=en) that explains how to configure remote storage using Fastly.

Hey!

I think the support part is about getting support from the cloud support team on your remote storage location. By enabling remote storage, you move the storage of media files to a private location. Thus the support team cannot access your media files.

If you could get someone from the product or cloud team to review, that be great. We implemented this on a customer's project and found a lack of documentation. Hence we added it.

hguthrie commented:
@robin31  I am on the Cloud team. We are reviewing. Thanks!

hguthrie reviewed (approved):
Okay, I am importing this and will process it internally. Thanks for the PR!
